examples/wrappers/go/generate_tensorflow_proto.sh

```diff
@@ -1,8 +1,15 @@
-git clone https://github.com/tensorflow/tensorflow.git
+
+DEST_REPO="tensorflow"
+REPO="https://github.com/tensorflow/tensorflow.git"
+if [ -d "${DEST_REPO}" ]; then
+    git -C ${DEST_REPO} pull || true
+else
+    git clone ${REPO} ${DEST_REPO}
+fi
```

Comment:
For the sake of consistency, it would be good to follow something similar to what we do for the Python wrapper: https://github.com/SeldonIO/seldon-core/blob/8160636472cd3ecdc23b76ff6fe981ceba4bf222/python/Makefile#L4-L11

That way we also can reuse the existing code under the `/proto/` folder to fetch the latest versions of the protobufs.